harvard-huit/Harvard-Geospatial-Library
# Index map Harvard note field enhancements

**Index map Harvard note field enhancements**
* * *

# What does this Pull Request do?
* made linkfyNote() javascript more flexible and will now remove any text that comes before the button (was previously relying on the text 'View in library: ')
* changed button language from "Request this item through HOLLIS" to "Request to scan or visit"

# How should this be tested?
* Harvard index map buttons will now say "Request to scan or visit" ([local index map example](https://localhost:3001/catalog/harvard-g3200-2000-g5-tpc-index-gr))
* Any text the might appear before the button (like View in library) will not appear
* Note fields for non-Harvard index maps will not be affected by this change, though I cannot find an example of a non-Harvard index map with a note field to confirm this. (Marc is looking into making a fake record for us to test this with)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no

# Interested parties
@phil-plencner-hl 
@enriquediaz
